buildenv/jenkins/variables/defaults.yml

```diff
@@ -341,12 +341,18 @@ x86-64_linux:
     8: 'linux-x86_64-normal-server-release'
     11: 'linux-x86_64-normal-server-release'
   node_labels:
-    build: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+    build:
+      all: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
+      8: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      11: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
   extra_configure_options:
     all: '--enable-jitserver'
   build_env:
     cmd: 'source /home/jenkins/set_gcc7.5.0_env'
     vars: 'OPENJ9_JAVA_OPTIONS=-Xdump:system+java:events=systhrow,filter=java/lang/ClassCastException,request=exclusive+prepwalk+preempt'
+  extra_test_labels:
+    15: '!sw.os.cent.6'
+    next: '!sw.os.cent.6'
```

Comment:
This is currently setup to be defined based on (test) target (not jdk version). See [variables-functions.groovy#L683](https://github.com/eclipse/openj9/blob/8ae17a7bb923a921d9435247636951141d0ea375/buildenv/jenkins/common/variables-functions.groovy#L683). However, there are currently only 2 known usages (to my knowledge). The first, applies an extra label without defining a target, therefore applying it to all targets (see internal variable file L174) and the second use case is no longer needed. Therefore I think we should update L683 to use `SDK_VERSION` instead of `target`. Also see infra issue 3914.